footer/main.scss

```diff
@@ -12,8 +12,8 @@
 }
 
 .o-footer__more-from-ft .o-icons-icon--arrow-right {
-  background-image: url("https://www.ft.com/__origami/service/image/v2/images/raw/fticon:arrow-right?source=o-icons&tint=%23FFFFFF,%23FFFFFF&format=svg");
-  background-image: url("https://www.ft.com/__origami/service/image/v2/images/raw/fticon:arrow-right?source=o-icons&tint=%23FFFFFF,%23FFFFFF&format=png&width=16") \9;
+  background-image: url("https://www.ft.com/__origami/service/image/v2/images/raw/fticon:arrow-right?source=alphaville2&tint=%23FFFFFF&format=svg");
```

Comment:
The source parameter is meant to be your products systemcode, this ensures that we know which products are using our service. Knowing which products are making specific requests helps us isolate issues and prioritise our workload.